homeassistant/components/cloud/__init__.py

```diff
@@ -136,12 +137,21 @@ def is_cloudhook_request(request):
     else:
         kwargs = {CONF_MODE: DEFAULT_MODE}
 
+    # Alexa/Google custom config
     alexa_conf = kwargs.pop(CONF_ALEXA, None) or ALEXA_SCHEMA({})
     google_conf = kwargs.pop(CONF_GOOGLE_ACTIONS, None) or GACTIONS_SCHEMA({})
 
+    # Cloud settings
     prefs = CloudPreferences(hass)
     await prefs.async_initialize()
 
+    # Cloud user
+    if not prefs.cloud_user:
+        user = await hass.auth.async_create_system_user(
+            'Home Assistant Cloud', [GROUP_ID_ADMIN])
```

Comment:
Maybe `GROUP_ID_USER`?

Reply:
I think that we should first see how our permissions will work out before I would want to consider that. The downside of the user group is that things like stop/restart HA will not be accessible.

Reply:
I am not a Cloud user, but in my assumption of HA Cloud function, this user is reserved for the Alex/GA or other cloud web hook integration purpose. If user want to do admin task, such as restart, he could remote login via Cloud by using the administer creds

Reply:
Well, the user rights are mapped local in Home Assistant and not inside the cloud. That means there is all transparent and controlled by local instance and not by external cloud.